public/app/redux/reducers/continuous/index.ts

```diff
@@ -72,27 +74,34 @@ const initialState: ContinuousState = {
   },
 };
 
-export const reloadAppNames = createAsyncThunk(
-  'names/reloadAppNames',
-  async (_, thunkAPI) => {
-    // TODO, retries?
-    const res = await fetchApps();
-
-    if (res.isOk) {
-      return Promise.resolve(res.value);
-    }
-
-    thunkAPI.dispatch(
-      addNotification({
-        type: 'danger',
-        title: 'Failed to load app names',
-        message: res.error.message,
-      })
-    );
-
-    return Promise.reject(res.error);
+export const reloadAppNames = createAsyncThunk<
+  App[],
+  null,
```

Comment:
If you replace this `null` with `void` you can revert passing `null` in the dispatch calls to `reloadAppNames` above.

Reply:
Awesome! This is exactly what I was trying to do yesterday but couldn't figure it out. This also means [this PR](https://github.com/grafana/pyroscope-app-plugin/pull/86) is not relevant anymore. Fixed in https://github.com/grafana/pyroscope/pull/2482/commits/4e932cc9c6edab2db9dae299819ec527dd1acfbe and https://github.com/grafana/pyroscope/pull/2482/commits/d2012bd2c6cd546bab146bd47bb43eb811e6c672.